habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -558,16 +558,19 @@ def action_space(self):
             }
         )
 
-    def step(self, human_joints_trans, is_last_action, **kwargs):
+    def step(self, *args, is_last_action, **kwargs):
         r"""
         Updates the joint rotations and root transformation of the humanoid.
-        :param human_joint_trans: Array of size (num_joints*4)+16. The last 16
+        :param human_joints_trans: Array of size (num_joints*4)+16. The last 16
```

Comment:
according to line 571, this is not the correct parameter name.

Reply:
Sorry, I don't understand this comment. Do you mean I should remove this parameter, and only document `is_last_action`?

Reply:
I mean that the correct parameter name is `self._action_arg_prefix + "human_joints_trans"`